lib/create-plugin.js

```diff
@@ -382,8 +369,9 @@ function getDeepestUniqueSelection(definition) {
       const lastItemIdx = deepestPath.length - 1
       // add type to the last item in deepestPath array
       // (i.e - `_entities<Human>`)
-      deepestPath[lastItemIdx] =
-        `${deepestPath[lastItemIdx]}<${selection.typeCondition.name.value}>`
+      deepestPath[
+        lastItemIdx
+      ] = `${deepestPath[lastItemIdx]}<${selection.typeCondition.name.value}>`
```

Comment:
eww. sometimes prettier has some odd ideas.

is this changeable or another thing they force on you?

Reply:
i can look but this is consistent with other areas where there are nested variable access, similar to args to a function

Reply:
I refactored this to break down the assignment.  What's happening here is the line is in violation of the `printWidth: 100`. So prettier then starts breaking it down and it always puts variables on new lines with indents like this

Reply:
odd given that is already indented to avoid being over 100. its a pretty horrible auto refactor so hopefully it doesn't do that to us much.